sdk/monitor/azquery/cloud_config.go

```diff
@@ -0,0 +1,41 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azquery
+
+import "github.com/Azure/azure-sdk-for-go/sdk/azcore/cloud"
+
+const (
+	MonitorQueryLogs    cloud.ServiceName = "azqueryLogs"
```

Comment:
These will need doc comments.  In addition, I think this is the first time we've added constants related to cloud config (outside of `azcore`), so we don't have a well-established naming pattern.

Reply:
CC @JeffreyRichter

Reply:
@chlowell Do we want to standardize the way we name the cloud.ServiceName? Also, I have `MonitorQueryLogs` and `MonitorQueryMetrics` declared in `azquery`, should we move it to the `cloud` package? That's where the arm cloud.ServiceName is declared.  
https://github.com/Azure/azure-sdk-for-go/blob/80bf31d4cb0db751479079cc4a1718adc7c8b547/sdk/azcore/cloud/cloud.go#L28

Reply:
We definitely want them in `azquery`.  The thinking is that modules would declare these constants and add them to the global config as you did below.  That way, new modules don't have to update `azcore/cloud` with their constants (which might also introduce some naming complexities if they're all in the same package).

So really the question is, what's the naming pattern we want to establish for these per-module, cloud-config constants?

Reply:
If we move service names into the cloud pkg, any new service updates cloud. This is perhaps OK if no other pkg takes a dependency on the cloud pkg.

Reply:
ServiceName is a type like an enum type, right? So, we would name them like we name enums: ServiceNameResourceManager. ServiceNameMonitorQueryLogs?

I know they're long but what are you gonna do?

Reply:
Good point about `ServiceName` being an enum type, so I agree we should follow the convention.  I would then suggest that we shorten the names.  Maybe something like this.
```go
ServiceNameLogs cloud.ServiceName = "azqueryLogs"
```
Making the fully qualified type name be `azquery.ServiceNameLogs` etc.

Reply:
Works for me!